### ISSUE TITLE
Submit Button

### ISSUE THREAD

**Issue by cheboi** [Naitwaevans/Rating-scale#1]
When submitting the rating it should pop the thank you section, with the rate selected, (create a Modal for that).

**Comment by Naitwaevans** [comment]
Okay, I will retry and do the whole project afresh, because of the CSS, and the fact that I have to create a modal. Part ya js ni kidogo

**Comment by cheboi** [comment]
Not really, you don't have to redo the whole project, just work on the js part and small CSS. 

you can read this docs https://www.w3schools.com/howto/howto_css_modals.asp
